website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md

```diff
@@ -98,3 +98,34 @@ The global attributes in the Loom apply to the whole file, not any specific part
 |`molecules_with_single_read_evidence`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of molecules associated with this entity that are observed by only one read. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.CellMetrics.molecules_with_single_read_evidence)|
 |`number_cells_detected_multiple`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of cells which observe more than one read of this gene. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.GeneMetrics.number_cells_detected_multiple)|
 |`number_cells_expressing`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of cells that detect this gene. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.GeneMetrics.number_cells_expressing)|
+
+## HCA Data Coordination Platform Matrix Processing
+Optimus supports data processing for the Human Cell Atlas (HCA) Data Coordination Platform (DCP; see the DCP [Data Portal here](https://data.humancellatlas.org/)). 
+
+All DCP Projects processed with Optimus have matrices containing the standard metrics and counts detailed above, but also have additional post-processing to incorporate HCA metadata.
+
+**These matrices are minimally filtered so that only cells with 100 molecules or more are retained.**
+
+:::warning
+This section details matrices produced for the Human Cell Atlas (HCA) [Data Coordination Platform (DCP)2.0](https://data.humancellatlas.org/), which includes matrices processed with Optimus v4.1.7 and later. The DCP is currently reprocessing data generated with earlier Optimus versions and will deprecate previous matrices once reprocessing is complete. 
+:::
+
+HCA DCP matrices contain HCA metadata in the Loom global attributes (see table below) which may be useful when exploring the data and linking it back to the Project metadata. Read more about each metadata field in the HCA DCP [Metadata Dictionary](https://data.humancellatlas.org/metadata). 
+
+| Metadata Attribute Name in Count Matrix | Metadata Description | 
+| --- | --- |
+| `donor_organism.genus_species` | species information; human or mouse |
+| `library_preparation_protocol.library_construction_method` | technology used for library preparation, i.e 10x or SS2 |
+| `specimen_from_organism.organ` | organ |	
+| `project.project_core.project_name` | project name |
+| `project.provenance.document_id` | project id |
+| `input_id` | metadata values for  `sequencing_process.provenance.document_id`; unique ID to demarcate the library prep |
```

Comment:
We also have input_id_metadata_field and input_name_metadata_field

Reply:
@khajoue2 Are those being carried over into the HCA Project matrices?